governance.md

```diff
@@ -0,0 +1,101 @@
+## Mission & Scope
+
```

Comment:
Maybe a subheading "History", or similar here : or "Where did it came from". This one is rather opinionated and not likely to change, whereas the "mission" later may evolve in the future ?